lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -97,17 +97,30 @@ class CategoryRenderer {
 
   /**
    * @param {!ReportRenderer.CategoryJSON} category
+   * @param {!Object<string, !ReportRenderer.CertificationJSON>=} certDefinitions
    * @return {!Element}
    */
-  _renderCategoryScore(category) {
+  _renderCategoryScore(category, certDefinitions) {
     const tmpl = this._dom.cloneTemplate('#tmpl-lh-category-score', this._templateContext);
     const score = Math.round(category.score);
 
     const gaugeContainerEl = this._dom.find('.lh-score__gauge', tmpl);
     const gaugeEl = this.renderScoreGauge(category);
     gaugeContainerEl.appendChild(gaugeEl);
 
-    return this._populateScore(tmpl, score, 'numeric', category.name, category.description);
+    const element = this._populateScore(
+        tmpl, score, 'numeric', category.name, category.description);
+
+    if (certDefinitions && category.certification && category.isCertified) {
```

Comment:
someone once told me we should trust our closure types and not be defensive beyond them... ;)